conf/application.conf

```diff
@@ -35,9 +36,32 @@ microservice {
       host = localhost
       port = 9250
     }
+
+    auth-login-api {
+      host = localhost
+      port = 8585
+    }
+
+    auth {
+      host = localhost
+      port = 8500
```

Comment:
we need to update app-config-base for this service so that it doesn't point to local-host for these services - it'd be better to do that before the pipeline goes off so that the new config is picked up first time